Userland/Applications/CertificateSettings/main.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CertificateStore.h"
+
+#include <LibCore/System.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Icon.h>
+#include <LibGUI/SettingsWindow.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+ErrorOr<int> serenity_main(Main::Arguments args)
+{
+    TRY(Core::System::pledge("stdio rpath wpath cpath recvfd sendfd unix proc exec"));
```

Comment:
you never decrease the pledge here. are there any of these that we can drop once the settings window is created?

Reply:
As far as I tried, I cant drop any. Will test `proc` and `exec` again.

Reply:
Was able to drop both from the start.